### ISSUE TITLE
401 error

### ISSUE THREAD

**Issue by ShuaishuaiLv1** [ChrisWu1997/onshape-cad-parser#3]
Hello，
I copied and ran this code, but when I run process.py --test, I always encounter a 401 error. I checked and found that I can run the get_features file, but an error occurs when running featurelistparser. Can you help me solve this problem?
401 Client Error: Unauthorized for url: https://cad.onshape.com/api/v6/partstudios/d/3aaf088d40add9c4871c193d/w/a9450b3cd6ea2636f1cb50ed/e/a1fb2e1e5bd4ddc9845173b4/featurescript

**Comment by LJXCMQ** [comment]
I had the same problem as you, have you solved it yet?

**Comment by ShuaishuaiLv1** [comment]
I have resolved this issue. Initially, I was connecting to the server, which resulted in a 401 error. However, when I called the API locally, it worked.

**Comment by LJXCMQ** [comment]
Thank you for your answer, but I would like to know more details.
I found out during debug that the program builds the following URL: https://cad.onshape.com/api/partstudios/d/8c3b97c1382c43bab3eb1b48/w/43439c4e192347ecbf818421/e/ 63b575e3ac654545b571eee6/featurescrip?
But the response returned by the URL is 401.
May I ask if you are in the same situation as me?
Also, I don't quite understand what you describe as local and server, does server mean git or remote host, and does local end mean personal computer?

**Comment by ShuaishuaiLv1** [comment]
Yes, I encountered the same problem as you. The server is a remote host, and I resolved this issue by using my own computer to call the API.

**Comment by Zuozhuo** [comment]
python process.py --test 
00001616
('00001616', 'parse feature failed:', IndexError('list index out of range',))



Have you encountered this issue before? The environment was set up according to the README, so it shouldn’t be an environment problem.

**Comment by gaoqy99** [comment]
> python process.py --test 00001616 ('00001616', 'parse feature failed:', IndexError('list index out of range',))（'00001616'， '解析功能失败：'， IndexError（'列表索引超出范围'，））
> 
> Have you encountered this issue before? The environment was set up according to the README, so it shouldn’t be an environment problem.您以前遇到过这个问题吗？环境是根据 README 设置的，因此它应该不是环境问题。

May I ask if you have solved this problem?
